manifests/init.pp

```diff
@@ -93,8 +93,9 @@
     validate_absolute_path($ssl_cert_db, $ssl_cert_password_file)
   }
 
-  class { '::qpid::install': } ~>
-  class { '::qpid::config': } ~>
-  class { '::qpid::service': } ->
-  Class['qpid']
+  include ::qpid::install
```

Comment:
Why not contain all of the classes?

If I want to do a `require => Class['qpid']` to ensure the package is installed, this still suffers from the same issue.